### ISSUE TITLE
Need help organizing page options

### ISSUE THREAD

**Issue by danyj** [ThemeFuse/Unyson#1352]
@moldcraft  so far I got most of the theme options in , now I am on page options and I cant seem to figure out how to set them up. 

Example , when you install theme you have these footer options

![screenshot_1](https://cloud.githubusercontent.com/assets/2340172/13722682/12134ea0-e84e-11e5-957c-e3a757649cbc.jpg)

than in a page settings you have this 

![screenshot_5](https://cloud.githubusercontent.com/assets/2340172/13722714/c8415d7a-e84e-11e5-9b1c-ad82a2c84a43.jpg)

when opened 

![screenshot_6](https://cloud.githubusercontent.com/assets/2340172/13722718/dd86e506-e84e-11e5-9fb4-2b2549ffa997.jpg)

the problem , 

lets say you leave page options at theme default , you setup your page and you done , 
than you decide to turn off socials trough theme options

![screenshot_4](https://cloud.githubusercontent.com/assets/2340172/13722693/5f24a694-e84e-11e5-865c-220a4b25976b.jpg)

now if you go back to the saved page and open the multi , it will tell you that the 

socials are enabled because you saved the page before you turned the socials off in theme options 

how can I set this up that before multi is opened the page options always show theme defaults ?

I do have these as default values 

    'value' => fw_get_db_settings_option('footer_socials_switch', 1),

but that does not help because once page is saved the multi opened or closed saves the page options

basically , how can I make the multi  save the options only when I select  

    Set custom options

any help or suggestion to set those up right is welcome

**Comment by moldcraft** [comment]
1. Add in Theme Settings options

    ```php
    'theme-option-id' => array(
        'type' => 'switch',
    ),
    ```

2. Add in Post options

    ```php
    'post-option-id' => array(
    	'type' => 'switch',
    	'fw-storage' => array(
    		'type' => 'wp-option',
    		'wp-option' => 'fw_theme_settings_options:'. fw()->theme->manifest->get_id(),
    		'key' => 'theme-option-id',
    	),
    ),
    ```
